internal/codespaces/api/api.go

```diff
@@ -556,6 +557,50 @@ func (a *API) GetCodespaceRepoSuggestions(ctx context.Context, partialSearch str
 	return repoNames, nil
 }
 
+// GetCodespacePreFlight returns the billable owner and expected default values for
+// codespaces created by the user for a given repository.
+func (a *API) GetCodespacePreFlight(ctx context.Context, nwo string) (*User, error) {
```

Comment:
Only comment is that preflight is a fairly established thing for browsers so when I personally see this it feels like it's doing quite a bit more than I'd expect. That might just be a me thing though!

Reply:
Sorry, mind clarifying? Is your point that the name "pre flight" confuses the intent here?